mirrord/operator/src/crd/policy.rs

```diff
@@ -78,6 +86,12 @@ pub struct MirrordClusterPolicySpec {
     // TODO: make the k8s list type be set/map to prevent duplicates.
     /// List of features and operations blocked by this policy.
     pub block: Vec<BlockedFeature>,
+
+    /// List of environment variables that should be excluded when using mirrord.
+    ///
+    /// These environment variables won't be retieved from the target even if the user
+    /// specifies them in their `feature.env.include` mirrord config.
+    pub env_vars_exclude: HashSet<String>,
```

Comment:
I think `excluded_env_vars` sounds better

Reply:
I agree, but we kinda use "env vars exclude" everywhere else :sweat: 

Like in the env config, and in all internal stuff we have (this one doesn't really matter though). I think this name ends up being more consistent, since `mirrord.json` navigation would also be more similar as it's `feature.env.exclude`.

wdyt? Am I overthinking this name?

Reply:
Perhaps to have a better UX it'd be better to make it closer to mirrord.json
so then we'd have `env` and under it `exclude` and perhaps later add include.

Reply:
That's a good UX idea, I'm also working on the file policies, think I should do the same thing there?

Reply:
Yes

Reply:
Changed.